components/brave_rewards/resources/page/components/pageWallet.tsx

```diff
@@ -35,8 +41,18 @@ interface State {
   modalQRCode: boolean
 }
 
+function constructBackupString (backupKey: string) {
+  return 'Brave Wallet Recovery Key\n' +
+    `Date created: ${new Date(Date.now()).toLocaleDateString()}\n\n` +
+    `Recovery Key: ${backupKey}\n\n` +
+    'Note: This key is not stored on Brave servers. This key is your only ' +
+    'method of recovering your Brave wallet. Save this key in a safe place, ' +
+    'separate from your Brave browser. Make sure you keep this key private, ' +
+    'or else your wallet will be compromised.'
+}
```

Comment:
Is it intentional that we're not localizing this?

Reply:
Removed.